### ISSUE TITLE
Make icons clearly visible at smaller sizes

### ISSUE THREAD

**Issue by StfBauer** [OfficeDev/office-ui-fabric-core#1076]
The icons provided by Office UI Fabric offer sadly a bad usability. The are drawn way to thin an special Icons like the question mark for help on Office 365 is hard to identify as a question mark.

Can you please take this feedback to design an check if they can come up with bitter icons. Customer of mine already complained too.

![screen shot 2017-12-03 at 19 42 50](https://user-images.githubusercontent.com/5503835/33622916-91559508-d9ef-11e7-8f01-6185c691ab66.png)

**Comment by Jahnp** [comment]
Hey @StfBauer, the Fabric team doesn't actually draw these icons--they're distributed from a centralized team based in the Windows group--but I'm happy to pass the feedback along to the team that does. Are you having trouble with the icons in general throughout Office 365, in your own apps/add-ins, or somewhere else?

**Comment by StfBauer** [comment]
@Jahnp I know. 👍 but you are the only one to address this when it comes to web.
The problem with such thin icons is that they doesn't scale that good on small sizes and are hardly to recognise.
The icons on the suite bar for example are currently **17px**. 
The smallest icons I screen-shoted icon on Windows are **20px - 22px**.
Below this size they are pretty hard to recognise plus on Windows they mostly companioned by a text.

To make a test I changed the font size first to 32px:
<img width="353" alt="screen shot 2017-12-05 at 20 36 59" src="https://user-images.githubusercontent.com/5503835/33627357-8ddac480-d9fd-11e7-970c-7c1adb898d98.png">

Changed it then to 20px:
<img width="176" alt="screen shot 2017-12-05 at 20 45 12" src="https://user-images.githubusercontent.com/5503835/33627384-a1d33a30-d9fd-11e7-80c6-9d84e144e5ae.png">

An these are the actual ones:
<img width="200" alt="screen shot 2017-12-05 at 20 48 59" src="https://user-images.githubusercontent.com/5503835/33627443-d2ac8b48-d9fd-11e7-8d38-e13aa2552e92.png">

With the current icons it is just a small difference but it certainly makes a difference which font-size you use. I also know there are many ways around this issue but I guess for a regular developer out there should be at least provided some guidance or maybe alternatively filled icons for smaller sizes.
